### PR TITLE
allow C's _Static_assert

### PR DESCRIPTION
**Pull request by kkourt**
Add an exception for _Static_assert in the CamelCase tests.

### CODE REVIEW

File: images/checkpatch/README.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ When building the Docker image, several patches are applied to the script.
   but not one it knows of, and it complains with a warning. This patch teaches
   it about the tag.
 
+* `fixes/ignore-_Static_assert.diff`: Allow Cilium and Tetragon developers to use `_Static_assert`,
+  even though it is is camel-cased.
```
Comment (qmonnet) [inline images/checkpatch/README.md:51]:
```suggestion
  even though it is camel-cased.
```

**Review by qmonnet — CHANGES_REQUESTED** [review]
Typo to fix in the README, but looks good otherwise. I don't see any solution for allowing `_Static_assert` other than patching the Perl script indeed.

**Comment by kkourt** [comment]
> Typo to fix in the README, but looks good otherwise. I don't see any solution for allowing `_Static_assert` other than patching the Perl script indeed.

Thanks! pushed typo, squashed, and pushed.

**Review by qmonnet — APPROVED** [review]
Thanks!

**Comment by kkourt** [comment]
@qmonnet do you permissions to merge this? (I do not)

**Review by NikAleksandrov — APPROVED** [review]

**Pull request merged**